app/renderer/src/styles/components/titlebar.ts

```diff
@@ -46,8 +46,6 @@ export const StyledMarkLogo = styled.img`
 
 	border-radius: 5px;
 	background-color: var(--color-bg-tertiary);
-
-	-webkit-app-region: no-drag;
```

Comment:
@roldanjr do you know if there is a reason this was set? It makes the icon draggable rather than acting as part of the draggable title bar.

Reply:
It's been so long, I already forgot it. If it solves the issue, I have no problem with that 😃  Thank you again @sekwah41